src/components/Header/Header.js

```diff
@@ -104,8 +104,14 @@ export class Header extends Component {
                   ) : (
                     ''
                   )}
-                  {!loggedIn ? this.renderNavItem('/login', 'Login') : ''}
-                  {!loggedIn ? this.renderNavItem('/register', 'Register') : ''}
+                  {!loggedIn ? (
+                    <Fragment>
+                      {this.renderNavItem('/login', 'Login')}
+                      {this.renderNavItem('/register', 'Register')}
+                    </Fragment>
+                  ) : (
+                    ''
```

Comment:
Can this be `null` instead of an empty string?

Reply:
Updated